src/mongo/tools/SConscript

```diff
@@ -15,9 +15,7 @@ env.SConscript(
 perconadecrypt = env.Program(
     target='perconadecrypt',
     source=[
-        'perconadecrypt.cpp',
-        'perconadecrypt_options.cpp',
-        'perconadecrypt_options.idl',
+        'perconadecrypt.cpp', 'perconadecrypt_options.cpp', 'perconadecrypt_options.idl',
         'perconadecrypt_options_init.cpp'
```

Comment:
I guess adding a comma `,` after `'perconadecrypt_options_init.cpp'` will make the formatter to place the previous three source code files each on its own line, as it was before.